src/components/ContextMenuItem.js

```diff
@@ -87,10 +87,12 @@ class ContextMenuItem extends Component {
                             }
                         >
                             {({hovered, pressed}) => (
-                                <Icon
-                                    src={icon}
-                                    fill={StyleUtils.getIconFillColor(getButtonState(hovered, pressed, this.props.isDelayButtonStateComplete))}
-                                />
+                                <View style={[StyleUtils.getWidthAndHeightStyle(16, 16), styles.alignItemsCenter, styles.justifyContentCenter]}>
```

Comment:
```suggestion
                                <View style={[StyleUtils.getWidthAndHeightStyle(variables.iconSizeSmall, variables.iconSizeSmall), styles.alignItemsCenter, styles.justifyContentCenter]}>
```

Please use variables.

Reply:
It looks like wrapping the icon in a view doesn't actually do anything. 

Passing `small` in the `<Icon>` component does though (your next suggestion). 

Why use the `<View>` wrapper?

Reply:
If we don't add a view with fixed width and height, the whole mini menu will shrink to fit the new icon size which is 16 against the old 20. So it will lose the purpose of reducing the icon size. 


We can change the Icon component and add props to pass down this style to the internal wrapper View. But it will break the component structure. The wrapper view in Icon https://github.com/Expensify/App/blob/04d0ab33df34a77457eacb412b46c4211aa70206/src/components/Icon/index.js#L63 is configured to take the size as per icon.

Reply:
> If we don't add a view with fixed width and height, the whole mini menu will shrink to fit the new icon size which is 16 against the old 20. So it will lose the purpose of reducing the icon size.

That's not what I'm seeing when I test it. Here they are side by side, rendered on web. Looks the same, right? 
![Screen Shot 2022-12-02 at 3 09 08 PM](https://user-images.githubusercontent.com/49007721/205404689-136b8a93-19ca-427c-8af9-4a07ac25daaf.png)
![Screen Shot 2022-12-02 at 3 10 24 PM](https://user-images.githubusercontent.com/49007721/205404694-b51d10b0-de1e-4c29-8de7-7a4cd471b216.png)

Reply:
Yeah, let me test.

Reply:
Excuse my silly code font. Some ppl barf 🤮 when they see it 😄

Reply:
https://github.com/tonsky/FiraCode

Reply:
Oh I've seen FiraCode! I'm a fan.

Reply:
Nice font ❤️